examples/imagewidget/iw_simple.py

```diff
@@ -0,0 +1,43 @@
+"""
+ImageWidget Simple
+============
+
+Example showing simple ImageWidget with pre-saved 512x512 random image.
+"""
+
+# test_example = true
+
+from fastplotlib import ImageWidget
+import numpy as np
+
+from wgpu.gui.offscreen import WgpuCanvas
+from pygfx import WgpuRenderer
+
+canvas = WgpuCanvas()
+renderer = WgpuRenderer(canvas)
+
+a = np.random.rand(500, 512, 512)
+
+data = np.load("../data/random3D.npy")
+
+iw = ImageWidget(
+    data=data,
+    slider_dims=["t"],
+    vmin_vmax_sliders=True,
+    cmap="gnuplot2"
+)
+
+data = np.load("../data/random.npy")
+
+iw.show()
+
```

Comment:
can you test before and after setting timepoint?

set timepoint using the slider  as well as `@current_index`

Reply:
need to think some more about testing image widget, somehow need renderer and canvas to be offscreen as in the other examples